.travis.yml

```diff
@@ -6,10 +6,10 @@ python:
   - "3.5"
   - "3.6"
 install:
-  - "pip install pytest-cov"
-  - "pip install -e ."
+  - "pip install -e .[test]"
 script:
   - "python -m pytest --cov mercantile --cov-report term-missing"
+  - "python -m pydocstyle mercantile"
```

Comment:
We're now running pydocstyle as a test that must be passed.

Reply:
I'm not sure docstrings are essential for test code. My theory is that the test name itself should be descriptive enough on its own.